corehq/apps/repeaters/models.py

```diff
@@ -132,6 +133,7 @@ class Repeater(QuickCachedDocumentMixin, Document, UnicodeMixIn):
     format = StringProperty()
 
     use_basic_auth = BooleanProperty(default=False)
+    auth_type = StringProperty(choices=("basic", "digest", ""))
```

Comment:
The user should be able to select "No authentication", which is here represented by an empty string. Do you think `StringProperty(choices=("basic", "digest"), required=False)` would be cleaner?